brew/src/test/java/nl/tudelft/serg/evosql/brew/db/EvoSQLRunnerTest.java

```diff
@@ -8,6 +8,7 @@
 import nl.tudelft.serg.evosql.fixture.FixtureTable;
 import nl.tudelft.serg.evosql.fixture.type.DBInteger;
 import nl.tudelft.serg.evosql.fixture.type.DBString;
+import nl.tudelft.serg.evosql.metaheuristics.DatabaseOutput;
```

Comment:
Since we're using `DatabaseOutput`s from `ga` again here, maybe we should create our own. However, it wouldn't be possible to directly convert between the two if we created one here too.

Reply:
What would you suggest?

Reply:
I think it's best to just stick with `List<HashMap<String, String>>` for now as it does not require a conversion then.

Reply:
Actually, if i'm correct, we can't remove this dependency since the original `PathResult` to which we pass the `DatabaseOutput` requires a type `DatabaseOutput` and not a `List<Map<String,String>>`, and since we then don't import the type by passing just a `List<Map<String,String>>` java won't know that that equals a `DatabaseOutput`.

I've tried this out and we unfortunately have to keep the dependency in there. I don't think it matters though since we have a lot of dependencies on that module here anyway.

Reply:
Hmm alright, I was wrong there, apologies. I agree with you on that one. Keeping it in here shouldn't be that much of an issue since we're already writing EvoSQL-specific code there.